unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Camera/CameraController.cs

```diff
@@ -28,6 +28,9 @@ public class CameraController : MonoBehaviour
         [SerializeField]
         internal InputAction_Measurable mouseWheelAction;
 
+        [SerializeField]
+        public OverrideCinemachineAxisInput overrideInput;
+
```

Comment:
Do we need this?

Reply:
Alright, will improve this with data stores